controllers/drpolicy_controller_test.go

```diff
@@ -439,6 +438,7 @@ var _ = Describe("DrpolicyController", func() {
 		It("should create a drcluster manifest work for each cluster specified in a 2nd drpolicy but not a 1st drpolicy",
 			func() {
 				drpolicyCreate(&drpolicies[1])
+				validatedConditionExpect(&drpolicies[1], metav1.ConditionTrue, Ignore())
```

Comment:
The race here was due to manifest getting created before the required secret artifacts are updated, causing an early check for secrets distribution placement and causing the spurious failures.

Reply:
Failure run for the above: https://github.com/RamenDR/ramen/runs/5408103750?check_suite_focus=true